gravitee-am-ui/src/app/utils/time-range-utils.ts

```diff
@@ -45,29 +45,29 @@ export const availableTimeRanges: TimeRange[] = [
   {
     id: '1d',
     name: 'Today',
-    value: 1,
-    unit: 'days',
+    value: 24,
+    unit: 'hours',
     interval: 1000 * 60 * 60,
   },
   {
     id: '7d',
     name: 'This week',
-    value: 1,
-    unit: 'weeks',
+    value: 7,
+    unit: 'days',
     interval: 1000 * 60 * 60 * 24,
   },
   {
     id: '30d',
     name: 'This month',
-    value: 1,
-    unit: 'months',
-    interval: 1000 * 60 * 60 * 24,
+    value: 4,
+    unit: 'weeks',
+    interval: 1000 * 60 * 60 * 24 * 7,
```

Comment:
**thought**: I'm wondering if we will have enough interval to make the graph useful :thinking:

Reply:
what do you propose?

Reply:
For a Month, I think a one day interval is good.